bridge-deploy/tests/test_cli.py

```diff
@@ -0,0 +1,54 @@
+import pytest
+
+from click.testing import CliRunner
+
+from bridge_deploy.cli import main
+
+
+@pytest.fixture
+def runner():
+    return CliRunner()
+
+
+def test_deploy_reward(runner):
+
+    result = runner.invoke(main, args="deploy-reward --jsonrpc test")
+
+    assert result.exit_code == 0
```

Comment:
shouldn't we also test here and in related functions that the contracts have been successfully deployed? E.g. by checking that there's code at the address? Or do we do this in other tests already?

Reply:
That's done in the test_deploy.py.

test_cli.py just check's that the CLI interface does "something" ;-)